### ISSUE TITLE
Add an RFC GitHub bot for Discord

### ISSUE THREAD

**Issue by phzarif85** [arionum-developers/rfc#2]
Can create new bot for this? So any new issue will automatically post in #to_do_list or any discord channel? So easily people see what news on here instead of need open github to view new issue.

**Comment by owenvoke** [comment]
I'll ask AroDev about this, Discord supports GitHub issue events so this would be easy to implement.

**Comment by owenvoke** [comment]
This is now ready, just waiting on AroDev for a webhook link. 👍

**Comment by owenvoke** [comment]
This is now implemented. :+1: Closing.

See the first post by RFC Bot on [Discord](https://discordapp.com/channels/399590985165963274/550076682296360971/579779596426412035).
